ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -113,6 +113,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-ipi-disc-priv-f28-ota
```

Comment:
Is OTA only supported on amd64 ? If not, shall we use arm64 ( or multi ) ?

Reply:
Currently the un-executed cases are all from osus which is amd64-only, so we choose amd64 to run disconnected cases.

Reply:
The other two - `gcp-ipi-ovn-ipsec` and `azure-ipi` are also including osus cases, right?

Reply:
right, and non-osus cases in other arch will be executed in regular ci, this job is just for the purpose in https://github.com/openshift/release/pull/55177#discussion_r1706282638